mobile/android/chrome/aboutAccounts.properties

```diff
@@ -0,0 +1,11 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+relinkDenied.message = Już zalogowano do Sync
+relinkDenied.openPrefs = USTAWIENIA
```

Comment:
This doesn't look right at all - are you sure "PREFS" (?!) is translated as "USTAWIENIA" in Polish Android UI?